app/uk/gov/hmrc/individualincomedesstub/domain/Validators.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualincomedesstub.domain
+
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.{Interval, LocalDate}
+
+import scala.util.Try
+
+object Validators {
```

Comment:
This looks more like a utility class than a domain object, hence it might be better placed in the `uk.gov.hmrc.individualincomedesstub.util` package.